INSTALL.md

```diff
@@ -75,18 +79,24 @@ Required Software:
 
```

Comment:
Could you please also update one line above
from:
scripts/mklfpk.sh [32|32e]

to 
scripts/mklfpk.sh